jobs/update-i18n-lite-videos.groovy

```diff
@@ -56,7 +56,7 @@ onMaster('23h') {
                    sender: 'I18N Imp',
                    emoji: ':smiling_imp:', emojiOnFailure: ':imp:',
                    extraText: "@cp-support",
-                   when: ['FAILURE', 'UNSTABLE', 'ABORTED']],
+                   when: ['SUCCESS', 'FAILURE', 'UNSTABLE', 'ABORTED']],
```

Comment:
I hope this is going to be one of those cases where Slack messages are not parsed just right, otherwise since it has `extraText: "@cp-support"` it might start generating CP support JIRA tickets :)

If that's the case, we'll need to drop the `@cp-support`.

Reply:
Found a failing example: https://khanacademy.slack.com/archives/C49296Q7P/p1666098440479329

It didn't generate a JIRA, but it would ping the on rotation `@cp-support` folks. So I think we still want to drop `extraText: "@cp-support",` for this reason if it's going to message on SUCCESS.

Reply:
@mpolyak I can go either way -- is it more importan to ping cp-support on failure, or to see the successes?  I'm guessing the former, so I'll just revert this change.

Reply:
Yeah, I think lets keep it the way it currently is.